src/Manifolds.jl

```diff
@@ -59,7 +60,6 @@ import ManifoldsBase: base_manifold,
     representation_size,
     retract,
     retract!,
-    check_tangent_vector,
```

Comment:
Why did you remove this?

Reply:
We can also add it again, I was thinking, the `is_tangent_vector` is now the interface (i.e. exported) and this one is internal, i.e. you can still call it with `Manifolds.check_tangent_vector` but the recommended way to check is the first mentioned method.

Reply:
Right, it makes more sense to not export it.